src/sentry/roles/manager.py

```diff
@@ -200,3 +200,10 @@ def with_any_scope(self, scopes: Iterable[str]) -> Iterable[OrganizationRole]:
 
     def get_minimum_team_role(self, org_role: str) -> TeamRole:
         return self.team_roles.get(self._minimum_team_role_map[org_role])
+
+    def get_sorted_organization_roles(self, org_roles: Iterable[str]) -> Iterable[OrganizationRole]:
+        return sorted(
+            [self.organization_roles.get(role) for role in org_roles],
+            key=lambda r: r.priority,
+            reverse=True,
```

Comment:
Nit: you can leave off `reverse=True` and write the key as `lambda r: -r.priority`. It's fine this way if you think it's more expressive, though -- matter of taste.